### ISSUE TITLE
no viable alternative at input '\n' - Window Line endings

### ISSUE THREAD

**Issue by LupusUmbrae** [dragos/typesafe-config-eclipse#8]
Just added this to my Eclipse (4.5.1) and when i open a config file more or less every line have this error `no viable alternative at input '\n'`, however if i change line endings from windows (CR + LF) to linux (LF) they disappear (instead i get `no viable alternative at input '<EOF>'` on the last line)

In the below example snippet the only line without the error would be `loglevel = "DEBUG"`, even the blank line has this error

```
akka {
  loglevel = "DEBUG"
  actor {
    # comment line

    # comment after blank line
```

**Comment by mkurz** [comment]
Same as #7?

**Comment by Mossinarium** [comment]
The `<EOF>` issue is the same as #7 but there is also an issue with Windows line endings what ever is parsing the file doesn't handle them thus the `no viable alternative at input '\n'`

**Comment by dragos** [comment]
Yes, the problem is most likely at [this line](https://github.com/dragos/typesafe-config-eclipse/blob/master/com.typesafe.hocon/src/com/typesafe/config/Hocon.xtext#L86). I am extremely busy for the next couple of weeks, but if you could help (seems like a simple one), I'd be more than happy to cut a new release.

**Comment by dragos** [comment]
Closed via #9
